pkg/applyconfigurations/testdata/cronjob/cronjob_types.go

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
Would it make sense to sync this with pkg/crd/testdata/cronjob_types.go?

I think the one in the CRD folder has a few more fields nowadays